src/main/java/com/googlesps/feedon/data/Donation.java

```diff
@@ -0,0 +1,37 @@
+// Class for donation offer made by restaurant
+package com.googlesps.data;
+
+import java.util.Date;
+import javax.annotation.Nullable;
+
+public final class Donation {
+    private final long id;
+    private final String restaurantName;
+    private final String location;
+    private final String category;
+    private final String pickUpTime;
+    private final String quantity;
+    private final String specialInstructions;
```

Comment:
Handling separate attributes instead of a list is more straight forward for now, wrt to the form submission, uploading to Datastore and front end rendering. But we sure can consider the structural change when we're refining the MVP!